tutorials/fundamentals/filter-key.md

```diff
@@ -0,0 +1,36 @@
+To filter an object using the keys, you can use JavaScript's `Object.hasOwnProperty()` method.
+
+```javascript
+const obj = { name: 'Masteringjs', location: 'Florida', help: true };
+let newObj = {}
+if(obj.hasOwnProperty('name')) {
+    newObj['name'] = obj['name'];
+}
+```
+
+## Object.keys()
+
+Another approach to filter using keys is to use the `Object.keys()` method.
+`Object.keys()` will return an array of the keys and you can then iterate through
+and filter accordingly.
+
+```javascript
+const obj = { name: 'Masteringjs', location: 'Florida', help: true };
+let newObj = {}
+let arr = Object.keys(obj);
+for(let i = 0; i < arr.length; i++) {
+    if (arr[i] == 'name') {
+        newObj[i] = obj[i];
+    }
+}
+```
+
+## Using One Variable
```

Comment:
I'd just get rid of this example because "Using One Variable" isn't a very clear description IMO.